lnwallet/btcwallet/psbt.go

```diff
@@ -59,24 +63,43 @@ func (b *BtcWallet) FundPsbt(packet *psbt.Packet, minConfs int32,
 		keyScope   *waddrmgr.KeyScope
 		accountNum uint32
 	)
+
+	// No change key scope should have been specified for custom accounts.
+	if changeScope != nil &&
```

Comment:
Maybe add another sentence here that because an account can (should) currently only have a single scope, changing the scope would break that assumption and allow the user to get into the non-deterministic situation by sending the change to a different scope than the account's defined scope.